src/Home/components/LecueBookList/index.tsx

```diff
@@ -19,40 +16,34 @@ interface LecueBookListProps {
 }
 
 function LecueBookList({ title }: LecueBookListProps) {
-  const navigate = useNavigate();
-  const deleteMutation = useDeleteFavorite();
   const isBookmark = title.includes('즐겨찾기');
   const { data } = isBookmark ? useGetFavorite() : useGetLecueBook();
 
-  const handleClickLecueBook = (uuid: string) => {
-    navigate(`/lecue-book/${uuid}`);
-  };
-
-  const handleClickFavoriteIcon = (bookId: number) => {
-    deleteMutation.mutate(bookId);
-  };
-
   return (
     <S.LecueBookListWrapper>
       <S.Title>{title}</S.Title>
       {data && data.length !== 0 ? (
         <S.LecueBookList>
           {data.map((book: BookProps) => (
             <S.LecueBook key={book.bookId} id={`${book.bookId}`}>
-              {isBookmark && (
-                <S.IconWrapper
-                  onClick={() => handleClickFavoriteIcon(book.bookId)}
-                >
-                  <IcHomeFavorite />
-                </S.IconWrapper>
+              {isBookmark ? (
+                <LecueBook
+                  bookId={book.bookId}
+                  bookUuid={book.bookUuid}
+                  favoriteImage={book.favoriteImage}
+                  favoriteName={book.favoriteName}
+                  bookType="favorite"
+                  deleteType="home"
+                />
+              ) : (
+                <LecueBook
+                  bookId={book.bookId}
+                  bookUuid={book.bookUuid}
+                  favoriteImage={book.favoriteImage}
+                  favoriteName={book.favoriteName}
+                  bookType="normal"
+                />
               )}
```

Comment:
ASK: 코드 실행해보니 deleteType 값이 home으로 고정되어도 mypage와 home 모두 잘 동작하는 것 같더라구요!
bookType 외에는 isBookmark 값에 따라 달라지는 부분이 없는 것 같은데, 혹시 제가 파악한 게 맞는 걸까용?

Reply:
P1: 앞선 질문에서 제가 파악한 게 맞다면, 중복코드를 최대한 줄이는 방향으로 수정하는게 좋을 것 같습니다!
```suggestion
                <LecueBook
                  bookId={book.bookId}
                  bookUuid={book.bookUuid}
                  favoriteImage={book.favoriteImage}
                  favoriteName={book.favoriteName}
                  bookType= isBookmark ? "favorite" : "normal"
                  deleteType="home"
                />
```

Reply:
헙 맞아요 isBookmark에 따라서 Home에서 즐겨찾기 한 레큐북인지, 인기 레큐북에 뜨는건지가 나뉩니다! deleteType은 Home과 Mypage를 구분짓는거라 요기서는 'home'으로 고정되어도 괜찮을 것 같습니다!! 안그래도 제가 생각해도 요기 코드가 별로였었는데 !! 아름이가 제안해준대로 바꿔볼게요 고마워요❤️